lib/src/Uncertainty/StatTests/VisualTest.cxx

```diff
@@ -334,29 +334,47 @@ GridLayout VisualTest::DrawPairsXY(const Sample & sampleX, const Sample & sample
 /* Draw 2-d projections of a multivariate sample, plus marginals of a distribution */
 GridLayout VisualTest::DrawPairsMarginals(const Sample & sample, const Distribution & distribution)
 {
+
   const UnsignedInteger dimension = sample.getDimension();
   if (dimension < 2)
     throw InvalidDimensionException(HERE) << "Can only draw clouds from a multivariate sample";
   if (distribution.getDimension() != dimension)
     throw InvalidDimensionException(HERE) << "Distribution dimension does not match the sample dimension";
   GridLayout grid(dimension, dimension);
   const Description description(sample.getDescription());
+  
+  const Scalar axesMargin = ResourceMap::GetAsScalar("VisualTest-DrawPairsMarginals-AxesMargin");
+  
+  const Point sampleMin = sample.getMin();
```

Comment:
Use `const Point sampleMin(sample.getMin());` as `Point` is a complex type. See the section named "Assignment and initialization" here https://openturns.github.io/openturns/latest/developer_guide/coding_rules.html